### DISCUSSION TITLE
2021-06-14 Meeting

### DISCUSSION THREAD

**Discussion by daxpryce** [microsoft/graspologic#799] [Meeting Notes]
Today's meeting broadly covered:
- Pipeline API changes
- @dlee0156's planned work to:
   - move autolayout into pipeline
   - de-privatize the functions that autolayout uses so they can be used as independent of the rest of the process
 - connected component utility enhancement #795 

Action items:
- Tutorials survey by a generalist, looking to ascertain whether the documentation is sufficient: @jonmclean 
- Hierarchical enhancements meeting: @dwaynepryce @bdpedigo 
- Diane (@dlee0156) to include Bryan Tower (@bryantower) and Dax (@dwaynepryce) (note: Dax needs to get the pipeline module kickstarted so we can place the auto layout stuff in there)
- Dax to create draft PR of azure functions around autolayout and leiden, then we can decide whether to do it in graspologic or make a new github repo for it or whatever.
- Ali (@asaadeldin11) has a PR for ASE/LSE in sklearn (these will work for csr matrices, right?)

**Comment by daxpryce** [comment]
Are there any current plans to get LSE into omni?

**Reply by bdpedigo** [reply]
https://github.com/microsoft/graspologic/pull/723

**Reply by daxpryce** [reply]
Oh right, true.  I think we'll need csr support in omni before we could even use LSE.  @Nyecarr brought up a good point in that we've never had strong justification on _why_ LSE needs to be in omni - it was just an ask from Jon and Carey afaik.

**Comment by daxpryce** [comment]
@dlee0156 when you make the layout changes, please put together a tutorial about using it.  You can probably use the autolayout as a good starting point for how to use it, but then maybe toss in a "oh but what if you already have your own embedding you want to use instead?  LOOK AT OUR UNBRIDLED MAJESTY bam" or something :D
